App/Redux/ThreadsRedux.ts

```diff
@@ -44,6 +62,15 @@ export type ThreadsState = {
     readonly threadId: string
     readonly error?: Error
   }
+  readonly links?: {
```

Comment:
Nit here and for `invites` below... we're just tracking a single invite and link at a time, so get rid of the plural wording?

Reply:
done